formats/json/commonMain/src/kotlinx/serialization/json/internal/lexer/JsonLexer.kt

```diff
@@ -131,6 +131,32 @@ internal class ReaderJsonLexer(
         preload(spaceLeft)
     }
 
+    override fun consumeStringChunked(consumeChunk: (stringChunk: String) -> Unit) {
+        consumeNextToken(STRING)
+        var currentPosition = this.currentPosition
+        var lastPosition = currentPosition
+        var char = source[currentPosition] // Avoid two range checks visible in the profiler
+        while (char != STRING) {
+            if (++currentPosition >= source.length) {
+                // end of chunk
```

Comment:
It seems that you're not handling string escape sequences. Is that intentional?

Reply:
Yes and no. For the first time, it was intentionally, because I suppose to consume base64 string, which can't contain double quote. But later, I prefer to generic approach, and seems, now I should handle double quotes as well. Will fix

Reply:
Added support for escaping

Reply:
Also, please note - to properly handle escaping, I'm forced to move actual decoding method from ReaderJsonLexer to AbstractJsonLexer. In other case I would need un-private a lot amount of methods of AbstractJsonLexer.

Reply:
Sorry, I don't quite understand. Can you elaborate on that please?

Reply:
Previously, I placed my method in ReaderJsonLexer to use highest possible hierarchy level. But to properly handle escaping I forced to move handling method to AbstractJsonLexer, go down one level. Is it OK?